File: problems/2739/main.cpp
// 2739. 구구단

#include <iostream>

int main()
{
    int number = 0;

    std::cin >> number;

    for (int multiplier = 1; multiplier < 10; ++multiplier)
    {
        std::cout << number << " * " << multiplier << " = " << number * multiplier << std::endl;
    }
}

File: problems/2739/main.rs
// 2739. 구구단

use std::io;

fn main() {
    let mut number = String::new();

    io::stdin()
        .read_line(&mut number)
        .expect("Failed to read line");

    let number: i32 = number.trim().parse().expect("Cannot parse number");

    for multiplier in 1..10 {
        println!("{} * {} = {}", number, multiplier, number * multiplier);
    }
}
